generators/frontend/templates/react-demo/components/Footer.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Link from 'next/link'
+import styled from 'styled-components'
+
+export default () => {
+  const Wrapper = styled.div`
+    color: darkslategrey;
+    padding: 30px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    hr {
+      border-top: 2px solid #f3f3f3;;
+      border-left-color: transparent;
+      height: 6px;
+      border-right-color: transparent;
+      border-bottom: 2px solid #f3f3f3;;
+      width: 74%;
+    }
+    div {
+      width: 100%;
+      .copyright {
+      font-size: 18px;
+      font-weight: 200;
+      color: darkslategrey;
+    }
+    .copyright {
+      margin-top: 50px;
+      font-size: 12px;
+      text-align: center;
+    }
+  }
+  `
+  const Navbar = styled.div`
+    margin: 30px auto 0 auto;
+    text-align: center;
+    a {
+      font-weight: 500;
+      color: darkslategrey;
+      margin: auto 50px 10px 50px;
+      text-decoration: none;
+    }
+  `
+  return(
+    <Wrapper>
+      <hr />
+      <Navbar>
+        <Link href={{ pathname: '/About', query: { name: 'about' } }}>
```

Comment:
You can generate it with for loop.
Crete the object with name and url and iterate here through it. Code will be much shorter and readable